en/docs/consuming-services/consuming-a-service-apikey.md

```diff
@@ -0,0 +1,83 @@
+# Consume an API Key Secured Service
+
+## Overview
```

Comment:
We don't really need this subheading since there is no content after the page heading.